docs/new_flux_ideas_and_guide.md

```diff
@@ -0,0 +1,985 @@
+minute or 1 hour or 1 day). There are different kinds of metrics that are commonly captured
+
+* counter - A value that counts the occurrence of an event. Counters alway increase unless they have 
+            been wrapped or reset. A reset occurs when what ever is doing the counting is restarted in 
+            which case it starts counting again from 0. A wrapped counter is one that has reached the 
+            largest number allowed by the byte representation of the value, in which case it starts over 
+            again at 0. These wraps and resets need to be accounted for when performing operations on counters. 
+            An example of a counter is the number of bytes a network interface has received. 
+* gauge - A value that measures a specific value, which can increase or decrease. An example of a gauge is 
+          taking a temperature reading.
+
+### Events
+Events are single data points in time. This could be an individual request to an API and how long that took, a user 
+logging into a service, a car parking, a light turning on, a container starting or stopping, or anything you can think 
+of. Events are what are called irregular time series given that they can occur at any time. It is possible to create 
+regular time series from an underlying event stream (like counting how many occurred every five minutes for the last hour).
+
+### Working with data in Flux
+When working with data in Flux, it translates the line protocol/time series structure into to a table. You 
+can think of a measurement name as a table with the tag and field keys representing columns. Flux also 
+includes the reserved column, `time`, in every table pulled from InfluxDB. The `time` column is the time at the end 
+of the line protocol when data is written in.
+
+Here's an example of line protocol and its translation to a Flux table:
+
+```
+h2o_temperature,location=santa_monica,state=CA surface_degrees=65.2,bottom_degrees=50.4 1568756160
+h2o_temperature,location=santa_monica,state=CA surface_degrees=63.6,bottom_degrees=49.2 1600756160
+h2o_temperature,location=coyote_creek,state=CA surface_degrees=55.1,bottom_degrees=51.3 1568756160
+h2o_temperature,location=coyote_creek,state=CA surface_degrees=50.2,bottom_degrees=50.9 1600756160
+h2o_temperature,location=puget_sound,state=WA surface_degrees=55.8,bottom_degrees=40.2 1568756160
+h2o_temperature,location=puget_sound,state=WA surface_degrees=54.7,bottom_degrees=40.1 1600756160
+```
+
+| location | state | surface_degrees | bottom_degrees | time |
+| --- | --- | --- | --- | --- |
+| coyote_creek | CA | 55.1 | 51.3 | 1568756160 |
+| coyote_creek | CA | 50.2 | 50.9 | 1600756160 |
+| puget_sound | WA | 55.8 | 40.2  | 1568756160 |
+| puget_sound | WA | 54.7 | 40.1  | 1600756160 |
+| santa_monica | CA | 65.2 | 50.4 | 1568756160 |
+| santa_monica | CA | 63.6 | 49.2 | 1600756160 |
+
+Note that the measurement name doesn't appear anywhere. That's because the table itself represents data from the 
```

Comment:
In my opinion, the first two sentences of this paragraph are the only ones needed here. At this point, i'm not thinking about multiple measurements or sorting, bringing that up here just takes away from the focus.

Reply:
Maybe add something like: Columns are created for tag keys, field keys, and time. Note that the measurement name...